coq.opam.template

```diff
@@ -0,0 +1,22 @@
+build: [
+  ["dune" "subst"] {dev}
+  [ "./configure"
```

Comment:
Maybe we can add a comment here in case we forget "TODO: remove configure call, which is only used to generate `config/Makefile`, instead do the call to generate `test-suite/config.inc` "